ui/apps/dashboard/src/components/Navigation/old/OrganizationDropdown.tsx

```diff
@@ -80,11 +80,15 @@ export default async function OrganizationDropdown() {
         </DropdownMenuGroup>
         <DropdownMenuGroup className="p-2">
           {organizations.length > 1 ? (
-            <OrganizationDropdownMenuItem
-              icon={RiArrowLeftRightLine}
-              href="/organization-list"
-              label="Switch Organization"
-            />
+            <DropdownMenuItem
```

Comment:
Also don't use a router link here as we want to get all new data when we switch orgs.